test/e2e/performanceprofile/functests/utils/hypershift/hypershift.go

```diff
@@ -22,6 +22,7 @@ const (
 	ManagementClusterKubeConfigEnv = "HYPERSHIFT_MANAGEMENT_CLUSTER_KUBECONFIG"
 	ManagementClusterNamespaceEnv  = "HYPERSHIFT_MANAGEMENT_CLUSTER_NAMESPACE"
 	HostedClusterKubeConfigEnv     = "HYPERSHIFT_HOSTED_CLUSTER_KUBECONFIG"
+	HostedClusterKubeConfigEnv     = "KUBECONFIG"
```

Comment:
in the commit message you say get rid, which I'd be fine it. Here you add a (duplicated) line. Bad rebase?

Reply:
Yea bad rebase